### DISCUSSION TITLE
"side by side" examples of LiaScript and quarto

### DISCUSSION THREAD

**Discussion by abotzki** [LiaScript/LiaScript#168] [Q&A]
Hi,

I am trying to make an example presentation which aims at a "side by side" comparison of the functionality of LiaScript and quarto presentations:

Here my first early draft - to the best of my capabilities in LiaScript ;-)
https://liascript.github.io/course/?https://raw.githubusercontent.com/vibbits/material-liascript/master/example-presentation.md

Here is the quarto "reference" - quarto has gained some traction in the R community - https://quarto.org/docs/presentations/revealjs/

We are planning to use this comparison as an example in our 2 day 'FAIR training material by design' course in September. First edition with be run in Uppsala, Sweden. Announcement to be opened in the coming days.

@andre-dietrich if you could help me fine-tune this example, I would be very, very grateful. Also happy to meet online if your time permits. Thanks a lot.

**Comment by andre-dietrich** [comment]
Hello @abotzki,

this sounds and looks great. Can you invite me to the project?

I figured out, that we screwed up intensely in the classroom functionality, which I try to repair at the moment ... The results look promising and I will make an update to LiaScript this week ... Probably the collaborative features with quizzes and questionnaires can be used as an highlight too?

Kind Regards,

André

**Reply by abotzki** [reply]
Hello @andre-dietrich - you should have an invitation (write access to the repo).

**Reply by andre-dietrich** [reply]
Just created a new branch, I called it ws/andre ... you can decide weather you want to merge it or take over some examples ...

You can directly see the examples here:

https://liascript.github.io/LiveEditor/?/show/file/https://raw.githubusercontent.com/vibbits/material-liascript/ws/andre/example-presentation.md

I added some more styling examples and another leaflet example, since it is implemented in JavaScript, it can be easily integrated ...

I added also more subsection ... maybe the empty slides can be filled with some nice gifs? This is what I mostly do in such situations ;-) ... otherwise simply move it back ;-)

https://giphy.com/

**Comment by SebastianZug** [comment]
@abotzki  According to the interesting idea of embedding jupyter notebook widgets in your [presentation](https://liascript.github.io/course/?https://raw.githubusercontent.com/vibbits/material-liascript/master/example-presentation.md#13) .... what would be necessary on the coderunner to make the example executable?

**Comment by abotzki** [comment]
@SebastianZug Here is the doc with examples https://quarto.org/docs/interactive/widgets/jupyter.html

Hopefully, this might help you further: https://ipywidgets.readthedocs.io/en/stable/embedding.html

Maybe, this is too much for most common users and only interesting for the more technical persons.
